ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/extensions/UriHandlerExtensions.kt

```diff
@@ -0,0 +1,16 @@
+package com.revenuecat.purchases.ui.revenuecatui.extensions
+
+import android.content.ActivityNotFoundException
+import androidx.compose.ui.platform.UriHandler
+
+/**
+ * Opens the given [uri] in a browser. If no browser is installed, [fallbackAction] is called.
+ */
+@Suppress("SwallowedException")
+fun UriHandler.openUriOrElse(uri: String, fallbackAction: () -> Unit) {
+    try {
+        openUri(uri)
+    } catch (e: ActivityNotFoundException) {
+        fallbackAction()
+    }
```

Comment:
Decided to go with try-catch instead of checking `PackageManager` for Activities that can resolve the intent, because this is an edge case. Otherwise the common case would also try to resolve the intent first, while it can just open it straight away.

Reply:
I think that's fine. This is better than crashing for sure